doc/project/checklist.md

```diff
@@ -411,6 +416,7 @@ The prefix "SIM" is applicable for simulation-based DV approaches, whereas the p
 
 It is possible for the design to have undergone some changes since the DV document and testplan were reviewed in the V0 stage.
 All design deltas have been captured adequately and appropriately in the DV document and the testplan.
+Double check with the designer that the correct parameterizations of the IP are being verified.
```

Comment:
IMO this should be a new paragraph for extra emphasis. How about this:
```
The testbench is setup to independently verify and close coverage on all parameter sets supported by the design (if applicable, this would have been documented under "Supported Parameter Sets" section in the design specification). 
```

Reply:
That makes sense. I've used your suggestion and re-pushed.